src/tests/ftest/daos_test/dfuse.py

```diff
@@ -91,8 +91,15 @@ def run_test(self, il_lib=None):
             daos_test_env['DD_MASK'] = 'all'
             daos_test_env['DD_SUBSYS'] = 'all'
             daos_test_env['D_LOG_MASK'] = 'INFO,IL=DEBUG'
-
-        command = [self.daos_test, '--test-dir', mount_dir, '--io', '--stream']
+            if il_lib == 'libpil4dfs.so':
+                daos_test_env['D_IL_REPORT'] = '0'
+                daos_test_env['DAOS_MOUNT_POINT'] = f'{mount_dir}'
+                daos_test_env['DAOS_POOL'] = f'{self.pool.identifier}'
+                daos_test_env['DAOS_CONTAINER'] = f'{self.container.uuid}'
+                daos_test_env['D_IL_MAX_EQ'] = '2'
+                daos_test_env['D_IL_ENFORCE_EXEC_ENV'] = '1'
```

Comment:
This is not a new test, what has changed to mean these are required now?

Reply:
The new tests in dfuse_test will check libpil4dfs related env to make sure they are enforced to set in child process.